.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,26 +13,26 @@ about: Create a report to help us improve
 
 ---
 
-## :writing_hand: Describe the bug
+## Description
 <!-- A clear and concise description of what the bug is. -->
 
-## :bomb: Steps to reproduce
+## Environment
+- **Device**: <!-- e.g. Pixel 5 -->
+- **OS**: <!-- e.g. 9.0.0 -->
+- **Software Information**: <!-- Add software information (e.g. name, version, build number, etc.). -->
```

Comment:
"Library version" seems more appropriate in our case. It was made generic like this because of other platforms (e.g., browser info in JS), but I don't see a reason for us to stick to the generic term.